command/server/config/config.go

```diff
@@ -69,6 +69,7 @@ type TxPool struct {
 	PriceLimit         uint64 `json:"price_limit" yaml:"price_limit"`
 	MaxSlots           uint64 `json:"max_slots" yaml:"max_slots"`
 	MaxAccountEnqueued uint64 `json:"max_account_enqueued" yaml:"max_account_enqueued"`
+	GossipBatchSize    uint64 `json:"gossip_batch_size" yaml:"gossip_batch_size"`
```

Comment:
Initially I did it like you suggesting, but at the end decided to go this way because we actually setting txpoolconfig which at the bottom level would look like defaultConfig.TxPool.TxGossipBatchSize, so that seemed redundant to me. So I don't know, though looking from outside (from the command perspective) it makes sense to add prefix tx.

Reply:
Done